aggie-coding-club/Rev-Registration
# Added sort by text and sort by shrinking

## Description

This adds sort by text in the format of `SORT: {type}`. It also makes the sort by text shrink if the row is running out of space.

Likewise, it forces the select all and sort by text to be on the same row when it runs out of space.

## How to test

Just open any course card and grow and shrink it and ensure it cuts off correctly.

## Screenshots

|Large|Small (cutoff)|
|--|--|
|![chrome_UHdDqd0Tnj](https://user-images.githubusercontent.com/7295783/114317144-8ff9b280-9abb-11eb-8751-f77686432e3c.png)|![chrome_WFpopm3UFb](https://user-images.githubusercontent.com/7295783/114317152-96882a00-9abb-11eb-9d8d-b0eed148bd3f.png)|


## Related tasks

Closes #519
